server/proof_fetcher_test.go

```diff
@@ -99,12 +99,12 @@ func TestTree32InclusionProofFetchAll(t *testing.T) {
 				refProof := mt.PathToRootAtSnapshot(int64(l+1), int64(s))
 
 				if got, want := len(proof.Hashes), len(refProof); got != want {
-					t.Fatalf("(%d, %d, %d): got proof len: %d, want: %d: %v\n%v", ts, s, l, got, want, fetches, refProof)
+					t.Fatalf("(%d, %d, %d): got proof len: %d, want: %d: %v\n%v", ts, s, l, got, want, nodes, refProof)
 				}
 
 				for i := 0; i < len(proof.Hashes); i++ {
 					if got, want := hex.EncodeToString(proof.Hashes[i]), hex.EncodeToString(refProof[i].Value.Hash()); got != want {
-						t.Fatalf("(%d, %d, %d): %d got proof node: %s, want: %s l:%d fetches: %v", ts, s, l, i, got, want, len(proof.Hashes), fetches)
+						t.Fatalf("(%d, %d, %d): %d got proof node: %s, want: %s l:%d nodes: %v", s, ts, l, i, got, want, len(proof.Hashes), nodes)
```

Comment:
What was the reasoning to swap `ts` and `s` around here (I think the original ordering was based on the nested loop vars: outer loop is `ts`, then `s`, then `l`) ?

Reply:
Restored the original order.